eth/gasprice.go

```diff
@@ -146,7 +150,7 @@ func (self *GasPriceOracle) processBlock(block *types.Block) {
 
 // returns the lowers possible price with which a tx was or could have been included
 func (self *GasPriceOracle) lowestPrice(block *types.Block) *big.Int {
-	gasUsed := new(big.Int)
+	gasUsed := common.Big0
```

Comment:
This really should stay `new(big.Int)`. This is error prone (e.g. you mistakenly modify `gasUsed` and you'll actually change everything that depend on it).

Use `common.Big0` as a constant, not as an initialiser.